src/dynamics/video_player/player/states.js

```diff
@@ -713,6 +713,7 @@ Scoped.define("module:VideoPlayer.Dynamics.PlayerStates.PlayVideo", [
 
         _started: function() {
             this.dyn.set("autoplay", false);
+            if (this.dyn.get("start_content_on_pause")) this.dyn.pause();
```

Comment:
Hi @Josewd! Looks good to me, the code is straightforward and easy to understand. 🙂
I'd just rename the variable to `pause_content_after_start` for clarity and move the part where we're disabling it to here, it feels a bit out of place inside the `pause` method.
```suggestion
            if (this.dyn.get("pause_content_after_start")) {
                this.dyn.pause();
                this.dyn.set("pause_content_after_start", false);
            }
```